### PR TITLE
use environment markers to specify enum34 requirement to fix pip install

### LINKED ISSUE #443: 0.11.0 pip missing package enum
Hi,

installing 0.11.0 via pip on python 2.7.X skips the enum package installation.

Tested on Ubuntu 16.04 LTS and CentOS 7.5

```
test$ bin/python -V
Python 2.7.12
```

```
test$ bin/pip install flask_restplus
Looking in indexes: <artifactory>/api/pypi/pypi/simple
Collecting flask_restplus
  Downloading ...
Collecting jsonschema (from flask_restplus)
  Downloading ...
Collecting aniso8601>=0.82 (from flask_restplus)
  Downloading ...
Collecting pytz (from flask_restplus)
  Downloading ...
Collecting six>=1.3.0 (from flask_restplus)
  Downloading ...
Collecting Flask>=0.8 (from flask_restplus)
  Downloading ...
Collecting functools32; python_version == "2.7" (from jsonschema->flask_restplus)
Collecting Jinja2>=2.10 (from Flask>=0.8->flask_restplus)
  Downloading ...
Collecting itsdangerous>=0.24 (from Flask>=0.8->flask_restplus)
Collecting Werkzeug>=0.14 (from Flask>=0.8->flask_restplus)
  Downloading ...
Collecting click>=5.1 (from Flask>=0.8->flask_restplus)
  Downloading ...
Collecting MarkupSafe>=0.23 (from Jinja2>=2.10->Flask>=0.8->flask_restplus)
Installing collected packages: functools32, jsonschema, aniso8601, pytz, six, MarkupSafe, Jinja2, itsdangerous, Werkzeug, click, Flask, flask-restplus
Successfully installed Flask-1.0.2 Jinja2-2.10 MarkupSafe-1.0 Werkzeug-0.14.1 aniso8601-3.0.0 click-6.7 flask-restplus-0.11.0 functools32-3.2.3.post2 itsdangerous-0.24 jsonschema-2.6.0 pytz-2018.4 six-1.11.0
```

### PR DESCRIPTION
**Pull request by Fearxpc**
https://www.python.org/dev/peps/pep-0508/#environment-markers

fix #443

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -59,8 +59,6 @@ def pip(filename):
 exec(compile(open('flask_restplus/__about__.py').read(), 'flask_restplus/__about__.py', 'exec'))
 
 install_requires = pip('install')
-if sys.version_info < (3, 4):
-    install_requires += ['enum34']
```
Comment (TotempaaltJ) [inline setup.py]: Does the `pip()` function not catch this? Weird...

Comment (Fearxpc) [reply setup.py]: Why should pip() catch it? There is no reference between the method and these two lines.

Comment (TotempaaltJ) [reply setup.py]: The pip function call in line 61 parses the `requirements/install.pip`, in which you've added `enum34` as a requirement.

Comment (TotempaaltJ) [reply setup.py]: 🤦‍♂️ I just realized these are removal lines and not additions... Never mind 😅

**Comment by noirbizarre** [comment]
Thanks, it's way better this way :ok_hand:

**Pull request merged**